multigen/worker_base.py

```diff
@@ -95,11 +95,6 @@ def close_session(self, session_id):
     def queue_gen(self, **args):
         self.logger.info("REQUESTED FOR QUEUE: " + str(args))
         with self._lock:
-            if args["session_id"] not in self.sessions:

```

Comment:
I'd keep these checks at least for logs. Or are they not relevant anymore at all?

Reply:
old api would work just fine. Relevance depends on the clients of our api